### ISSUE TITLE
Links that open in new tab are ignored/handed off to Chrome

### ISSUE THREAD

**Issue by Sub-Xaero** [responsively-org/responsively-app#612]
# 🐞 bug report

### ✍️ Description

I am aware that the app does not yet support tabs (#215), but the app also just completely ignores `a[target="_blank"]`. 
I would have thought that the app could at least open those links in the same tab.

Repeated clicking of the link also seems to open a new tab in an instance of Chrome, rather than in Responsively itself.

### 🕵🏼‍♂️ Is this a regression?

This is the first time I've used Responsively. I couldn't say.

### 🔬 Minimal Reproduction

Visit a page that has a link with `[target="_blank"]`, click the link.

### 🌍 Your Environment

<pre><code>Windows 10 - Build 19043.1083
Responsively v0.17.0
Google Chrome 91.0.4472.124 (Official Build) (64-bit) (cohort: stable_gradual_rollout)
</code></pre>

### 🔥 Exception or Error or Screenshot

<pre><code>N/A</code></pre>

**Comment by jjavierdguezas** [comment]
thanks @Sub-Xaero for report this

The expected behavior is to open a new window displaying the href (if it is a valid website url) or open the file if it is a local file and you are browsing on localhost. The fallback to all of this is to let the OS handle it, that's why you see Chrome open (I think we can improve the way URLs are validated, because if chrome can open it, we can too)

That worked for some time, but now I see that it doesn't (at least for some devices) 

https://user-images.githubusercontent.com/13673443/125442165-9812daf1-96fc-4d17-b51a-e5367e4cb8ea.mp4

**Comment by Sub-Xaero** [comment]
Makes sense. Thank you for looking into it @jjavierdguezas . The provided video matches my experience of the bug, I failed to pick up in my initial report that the behaviour was different depending which device the event was initiated from.
